### ISSUE TITLE
Some buttons have low contrast or seem to disappear in Dark Mode

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#31760]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When working in dark mode, some buttons disappear: 
-	the colour selection tool button (Preferences > Appearance > Background Colour) becomes invisible if the background colour is also black. 
-	Note head colour button (select note > properties > general > appearance > colour) 

The buttons need a contrasting outline, ideally one that is responsive and will change between light/dark mode.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.5.2

### Regression

I was unable to check

### Operating system

iOS: Sequoia 15.6

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
